packages/block-editor/src/store/private-selectors.js

```diff
@@ -547,6 +547,13 @@ export function isZoomOutMode( state ) {
 	return state.editorMode === 'zoom-out';
 }
 
+/**
+ * Retrieves the root client ID for a section from the state.
+ *
+ * @param {Object} state Editor state.
+ *
+ * @return {string|undefined} The root client ID for the section, or undefined if not set.
```

Comment:
```suggestion
 * @return {string|undefined} The root client ID or undefined if not set.
```

The description wasn't quite accurate. It's also explained above so I think we are ok to omit the extra detail 👍